### PR TITLE
Api improvements

### PR DESCRIPTION
**Pull request by Ozaq**

### CODE REVIEW

File: python_modules/jupedsim/jupedsim/native/models.py [hunk 1]
```diff
@@ -1,49 +1,32 @@
...
+@dataclass(kw_only=True)
+class GeneralizedCentrifugalForceModelParameters:
+    nu_ped: float
+    nu_wall: float
+    dist_eff_ped: float
+    dist_eff_wall: float
+    intp_width_ped: float
+    intp_width_wall: float
+    maxf_ped: float
+    maxf_wall: float
```
Comment (schroedtert) [inline python_modules/jupedsim/jupedsim/native/models.py:29]: Why no default parameters here?

File: python_modules/jupedsim/jupedsim/native/models.py [hunk 2]
```diff
@@ -1,49 +1,32 @@
...
 
 
 class GCFMModelAgentParameters:
```
Comment (schroedtert) [inline python_modules/jupedsim/jupedsim/native/models.py:32]: Every other occurrence of GCFM has been renamed to full name, except this part. We should be consistent in the naming here.

**Review by schroedtert — COMMENTED** [review]
One larger issue that i got: The trajectory data now only starts at frame 1. Which gives an empty image in the visualizer, as it starts at frame 0 but the agents only appear in frame 1 there.

**Comment by Ozaq** [comment]
> One larger issue that i got: The trajectory data now only starts at frame 1. Which gives an empty image in the visualizer, as it starts at frame 0 but the agents only appear in frame 1 there.

Fixed

**Pull request merged**